storm/src/main/storm/mesos/constraint/offer/NoOfferConstraint.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package storm.mesos.constraint.offer;
+
+import org.apache.mesos.Protos.Offer;
+import storm.mesos.constraint.Constraint;
+
+/**
+ * @author fuji-151a
```

Comment:
I have 3 reasons.
- MesosNimbus.java is very big class. Therefore the code readability is low.
So, I don't want to add code to MesosNimbus.java.
- Also MesosNimbus.java is big class, So It is hard to write testable code.
- I want to abstract this feature.
Because, not only An attribute value supports the attribute values of text type, but also scalar or range type.
For example, if you want to implement scalar or range type, you implement using [Constraint](https://github.com/fuji-151a/storm/blob/divide_class/storm/src/main/storm/mesos/constraint/Constraint.java) interface.